catalog/app/containers/Bucket/PackageList/PackageList.tsx

```diff
@@ -205,76 +207,101 @@ const useRevisionMetaStyles = M.makeStyles((t) => ({
     borderTop: `1px solid ${t.palette.divider}`,
     padding: t.spacing(2),
     ...t.typography.body2,
-    color: t.palette.text.secondary,
   },
   section: {
     '& + &': {
       marginTop: t.spacing(1),
     },
   },
-  tag: {
-    '& + &': {
-      marginLeft: t.spacing(1),
-    },
+  sectionWithToggle: {
+    marginLeft: '-5px',
   },
 }))
 
 interface RevisionMetaProps {
-  sections: (string | string[])[]
+  revision: SelectiveMeta
 }
 
-function RevisionMeta({ sections }: RevisionMetaProps) {
+function RevisionMeta({ revision }: RevisionMetaProps) {
   const classes = useRevisionMetaStyles()
-
   return (
     <div className={classes.root}>
-      {sections.map((section, i) => (
-        <div className={classes.section} key={`${i}+${section}`}>
-          {Array.isArray(section)
-            ? section.map((label, j) => (
-                <M.Chip
-                  className={classes.tag}
-                  label={label}
-                  key={`${j}+${label}`}
-                  size="small"
-                  variant="outlined"
-                />
-              ))
-            : section}
+      {revision.message && <div className={classes.section}>{revision.message}</div>}
+      {revision.userMeta && (
+        <div className={classes.section}>
+          {Object.entries(revision.userMeta).map(([name, value]) => (
```

Comment:
what's the point of showing every key as a separate object vs showing the object itself?

Reply:
For aesthetics

If showing the object itself, then we render two extra lines (collapse button + opening bracket and closing bracket)